package.json

```diff
@@ -40,7 +39,7 @@
     "jsdom": "3.0.0",
     "jshint": "^2.6.3",
     "react": "^0.14.0",
-    "react-router": "^1.0.3",
+    "react-router": "^2.4.0",
```

Comment:
@samsel and @remarkablemark here is what I get when I output the `err` variable for the callback of res.render in express:

Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined.
    Invariant Violation: Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined.
        at invariant ([project-path-removed]/node_modules/fbjs/lib/invariant.js:39:15)
        at instantiateReactComponent ([project-path-removed]/node_modules/react/lib/instantiateReactComponent.js:64:134)
        at [project-path-removed]/node_modules/react/lib/ReactServerRendering.js:41:31
        at ReactServerRenderingTransaction.Mixin.perform ([project-path-removed]/node_modules/react/lib/Transaction.js:136:20)
        at Object.renderToString ([project-path-removed]/node_modules/react/lib/ReactServerRendering.js:40:24)
        at renderAndDecorate ([project-path-removed]/node_modules/react-engine/lib/server.js:112:30)
        at reactRouterMatchHandler ([project-path-removed]/node_modules/react-engine/lib/server.js:178:31)
        at [project-path-removed]/node_modules/react-router/lib/match.js:58:5
        at [project-path-removed]/node_modules/react-router/lib/useRoutes.js:120:15
        at done ([project-path-removed]/node_modules/react-router/lib/AsyncUtils.js:49:19)